ttnn/cpp/pybind11/operations/data_movement.hpp

```diff
@@ -107,6 +109,33 @@ Example::
         )doc",
         ttnn::pybind_arguments_t{
             py::arg("input_tensor"), py::arg("shape"), py::kw_only(), py::arg("memory_config") = std::nullopt});
+
+    ttnn::bind_registered_operation(
+        module,
+        ttnn::repeat_interleave,
+        R"doc(
+repeat_interleave(input_tensor: ttnn.Tensor, repeats : int, dim: int = 0) -> ttnn.Tensor
```

Comment:
Important change to the inteface - `repeats` only accept `int`.
Before this change, the interface lies like it can support Tensor or Int, but in reality it only supported Int

Reply:
https://pytorch.org/docs/stable/generated/torch.repeat_interleave.html

We should probably flag this in a separate issue to note the difference relative to torch